src/tests/ftest/control/dmg_pool_query_test.yaml

```diff
@@ -27,8 +27,8 @@ ior:
 exp_vals:
   pool_status: 0
   pool_state: "Ready"
-  total_targets: 8
-  active_targets: 8
+  total_targets: 4
```

Comment:
Rather than fix it here you should probably just merge with master to pick up the fix